### PR TITLE
docs: add integration docs

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

docs: add integration docs

### CODE REVIEW

File: docs/integrate-logto/assets/icons/cloud-connection.svg
Comment (charIeszhao) [inline docs/integrate-logto/assets/icons/cloud-connection.svg:1]: Most of these icons can already be found in `/src/assets/` folder. And we need to manually modify the fill color to "currentColor", in order to make it dynamically apply the theme colors

File: docs/integrate-logto/interact-with-management-api/assets/logto-management-api-resource.png
Comment (charIeszhao) [inline docs/integrate-logto/interact-with-management-api/assets/logto-management-api-resource.png:1]: nit: Use webp format?

File: docs/integrate-logto/interact-with-management-api/README.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+- Resources
+- Connectors
+
+To learn more about the APIs that are available, please visit https://openapi.logto.io/.
+
+## How to access Logto Management API
+
+### Create an M2M app
+
+On the "Applications" page, select the M2M app type and start the creation process.
```
Comment (charIeszhao) [inline docs/integrate-logto/interact-with-management-api/README.md:45]: Use `<CloudLink to"/applications">Console > Applications</CloudLink>` instead?

File: docs/integrate-logto/interact-with-management-api/README.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+### Advanced user search
+
+The Logto Admin Console supports basic search and filtering functions. For advanced search options like fuzzy search, exact match, and case sensitivity, check out our [Advanced User Search](/docs/recipes/manage-users/advanced-user-search/) tutorials and guides.
+
+### Implement organization management on your own
+
+If you’re using the [organization](/docs/recipes/organizations/) feature to build your multi-tenant app, you might need the Logto Management API for tasks like organization invitations and member management. For your SaaS product, where you have both admins and members in the tenant, the Logto Management API can help you create a custom admin portal tailored to your business needs. Check out [this](/docs/recipes/organizations/configuration/#configure-via-management-api) for more detail.
```
Comment (charIeszhao) [inline docs/integrate-logto/interact-with-management-api/README.md:98]: These `/docs/recipes/...` links should be updated with new ones, as we are going to deprecate all new version docs later

File: docs/integrate-logto/third-party-application/oidc-third-party-application/permission-management.mdx [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+- `profile`: OIDC standard scope, used for accessing user name and avatar.
+- `email`: OIDC standard scope, used for accessing user email.
+- `phone`: OIDC standard scope, used for accessing user phone number.
+- `custom_data`: Logto user profile scope, used for accessing [user custom data](/user-management/user-data/#custom-data).
+- `identity`: Logto user profile scope, used for accessing user linked [social identities](/user-management/user-data/#social-identities) information.
+- `role`: Logto user profile scope, used for accessing user [role](/authorization/role-based-access-control) information.
+- `urn:logto:scope:organizations`: Logto user organization scope, used for accessing user organizations information. If enabled and requested by a third-party application, an organization selector will be displayed on the consent page. This allows users to review and choose the organization they wish to grant access to. See [organization](/organizations) for more details.
+- `urn:logto:scope:organization_roles`: Logto user organization scope, used for accessing user organization roles information.
+
+:::caution
```
Comment (charIeszhao) [inline docs/integrate-logto/third-party-application/oidc-third-party-application/permission-management.mdx:59]: `:::caution` is deprecated in docusaurus v3. Use `:::warning` instead

**Review by charIeszhao — APPROVED** [review]

File: docs/integrate-logto/interact-with-management-api/README.md (hunk 1 above)
Comment (xiaoyijun) [reply docs/integrate-logto/interact-with-management-api/README.md]: Updated.

File: docs/integrate-logto/interact-with-management-api/assets/logto-management-api-resource.png
Comment (xiaoyijun) [reply docs/integrate-logto/interact-with-management-api/assets/logto-management-api-resource.png:1]: Updated.

**Pull request merged**